test/libsolidity/GasCosts.cpp

```diff
@@ -104,9 +104,19 @@ BOOST_AUTO_TEST_CASE(string_storage)
 	else if (Options::get().useABIEncoderV2)
 	{
 		if (Options::get().optimizeYul)
-			CHECK_DEPLOY_GAS(151455, 127653, evmVersion);
+		{
+			if (evmVersion < EVMVersion::istanbul())
+				CHECK_DEPLOY_GAS(151455, 127653, evmVersion);
+			else
+				CHECK_DEPLOY_GAS(123, 113821, evmVersion);
```

Comment:
The `123` here is just not hit by the tests, right?

Reply:
Yes, marked them as 0 in a commit.